CONTRIBUTING.md

```diff
@@ -149,17 +148,17 @@ To build the packages, run the following commands:
 
 ```bash
 cd packages
-npm install
-npm run build
+yarn install
+yarn build
 ```
 
 To test a package locally, build the packages then run the following commands:
 
 ```bash
 cd lib/dist
-npm link
+yarn link
 cd /your/test/app
-npm link @h5web/lib
+yarn link @h5web/lib
```

Comment:
We might want to check if this still works. I seem to remember that linking with yarn was a bit different.

Reply:
It works the same way as NPM and gives the same "react hook call" issues 😂